pages/orders/[orderId]/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { useShippingAndProductsOrder } from '../../../lib/hooks';
+import { BillingAddress, OrderProduct, ShippingAndProductsOrder } from '../../../types';
+
+const StyledAddress = styled.address`
+    color: ${({ theme }) => theme.colors.secondary70};
+    font-style: normal;
+    line-height: ${({ theme }) => theme.lineHeight.small};
+`;
+
+const StyledUl = styled.ul`
+    ${({ theme }) => theme.helpers.listReset}
+
+    color: ${({ theme }) => theme.colors.secondary70};
+    line-height: ${({ theme }) => theme.lineHeight.xxLarge};
+`;
+
+const InternalOrderPage = (order: ShippingAndProductsOrder) => {
+    const { shipping_addresses = [], products = [] } = order
+    const items = shipping_addresses.map(address => {
+        const addressProducts = products.filter(({ order_address_id }) => order_address_id === address.id);
+
+        return { address, addressProducts }
+    });
+
+    const Address = (address: BillingAddress) => (
+        <StyledAddress>
+            <div>{address.first_name} {address.last_name}</div>
+            <div>{address.street_1} {address.street_2}</div>
+            <div>{address.city}, {address.state} {address.zip}</div>
+        </StyledAddress>
+    );
+
+    const renderOrderProducts = (addressProducts: OrderProduct[]) => (
+        <StyledUl>
+            {addressProducts.map(product => <li key={product.id}>{product.name}</li>)}
+        </StyledUl>
+    );
+
+    const renderOrderProductsQuantities = (addressProducts: OrderProduct[]) => (
+        <StyledUl>
+            {addressProducts.map(product => <li key={product.id}>{product.quantity}</li>)}
+        </StyledUl>
+    );
+
+    return (
+        <>
+            <Flex justifyContent="space-between" marginBottom="medium">
+                <H1>Order details</H1>
+                <Button>Create shipment</Button>
+            </Flex>
+            <Panel>
+                <StatefulTable
+                    columns={[
+                        { header: 'Ship to', hash: 'address', render: ({ address }) => <Address {...address} /> },
+                        {
+                            header: 'Products',
+                            hash: 'addressProducts',
+                            render: ({ addressProducts }) => renderOrderProducts(addressProducts),
+                        },
+                        {
+                            header: 'Quantity',
+                            hash: 'quantity',
+                            render: ({ addressProducts }) => renderOrderProductsQuantities(addressProducts),
+                        },
+                    ]}
+                    items={items}
+                /> 
+            </Panel>
+        </>
+    );
+};
+
+const OrderPage: React.FC = () => {
+    const router = useRouter();
+    const { orderId } = router.query;
+    const { isLoading, order, error }= useShippingAndProductsOrder(parseInt(`${orderId}`, 10));
+
+    if (isLoading) {
+        return <Loading />;
+    }
```

Comment:
🍹 these multiple line returns on basic conditionals are driving me nuts... lol we're only doing this on the orders pages and imho isn't consistent with the rest of the app. I think ideally we should do one or the other and be consistent.

```suggestion
    if (isLoading) return <Loading />;
```

Reply:
I'll make sure to include this in the linting update.

Reply:
thank you 🙏 😬